include/dfrn.php

```diff
@@ -16,6 +26,10 @@
  */
 class dfrn {
 
+	const DFRN_TOP_LEVEL = 0;
+	const DFRN_REPLY = 1;
+	const DFRN_REPLY_RC = 2;
```

Comment:
Should have a little comment. I had read some code until I get it that RC means remote contact